torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -198,9 +198,16 @@ std::pair<IValue, c10::optional<IValue>> getFunctionTuple(
 
   // operators
   std::vector<IValue> operators;
+  auto op_to_specified_args = code.op_to_num_specified_args();
   operators.reserve(opnames.size());
   for (const auto& opname : opnames) {
-    operators.emplace_back(Tup({opname.name, opname.overload_name}));
+    auto unique_name = c10::toString(opname);
+    int num_args = -1;
```

Comment:
Under what cases do we not expect to find an operator schema? Should this be an error if we always expect to find an operator schema?

Reply:
`op_to_specified_args` does not include operator with varargs. For those operators we don't do any default argument pushing. I'll add a comment on this.

Reply:
Which are the example operators with varargs? Presumably these are different from the ones with default args?

Reply:
Among > 2000 ops, there are 10s vararg operators where the number of argument varies. Those ops are different from default args and goes through OPN instruction.